Solver_interface/doc/Solver_interface/Concepts/SparseLinearAlgebraTraits_d.h

```diff
@@ -79,40 +81,45 @@ class SparseLinearAlgebraTraits_d::Vector
 */
 typedef unspecified_type NT;
 
+/*!
+Index type
+*/
+typedef unspecified_type Index;
```

Comment:
Shouldn't it be done at the traits level too?

Reply:
Actually, in the code is it only done at the traits level.

Reply:
Can we address that when we solve the more general Issue #3479 ?